vos/vos/commands/vls.py

```diff
@@ -125,7 +125,9 @@ def vls():
         else:
             sort = None
 
-        if opt.reverse:
+        if sort is None:
+            order = None
```

Comment:
I think argparse should display an error in this case and return: reverse cannot be used without sort.

Reply:
Updated. Thank you.

Reply:
Added a unit test case to test for this condition.